### PR TITLE
refactor(plugin): plugins automatically register and standard interfaces

### LINKED ISSUE #40: support plugin auto register to runner

### PR DESCRIPTION
**Pull request by shuaijinchao**
FIX #40

### CODE REVIEW

File: apisix/runner/server/handle.py [hunk 1]
```diff
@@ -16,7 +16,7 @@
 #
 
 import flatbuffers
-import apisix.runner.plugin.core as runner_plugin
+from apisix.runner.plugin.core import PluginProcess as RunnerPlugin
```
Comment (spacewander) [inline apisix/runner/server/handle.py:19]: Why not use the original name?

Comment (shuaijinchao) [reply apisix/runner/server/handle.py:19]: Uniform style with other names ^_^

Comment (spacewander) [reply apisix/runner/server/handle.py:19]:
```suggestion
from apisix.runner.plugin.core import PluginProcess as runner_plugin
```
would be more consistent

Comment (shuaijinchao) [reply apisix/runner/server/handle.py]: OK updated

**Review by spacewander — APPROVED** [review]

**Pull request merged**